.gitignore

```diff
@@ -178,12 +178,14 @@ zenml_examples/
 **wandb/
 !src/zenml/integrations/wandb
 
-
 # some examples have yamls; don't ignore them
 !examples/**/*.yaml
 
 # stack recipes
 test_terraform/
 
+# dashboard build files
+src/zenml/zen_server/dashboard/
+
```

Comment:
hehe :) we tried this before, but then the dashboard files are not included in the pypi package when we build it. I botched a release because of this.

Reply:
<iframe src="https://giphy.com/embed/afVwAjxEfqofu" width="480" height="323" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/reaction-the-office-no-afVwAjxEfqofu">via GIPHY</a></p>

Reply:
moving it into .git/info/exclude locally and removing it here